client-core/src/service/wallet_service.rs

```diff
@@ -101,11 +103,15 @@ pub struct WalletInfo {
     /// wallet meta data
     #[serde(deserialize_with = "deserde_from_str", serialize_with = "serde_to_str")]
     pub wallet: Wallet,
-    /// private key of the wallet
+    /// private key of view key
     #[serde(deserialize_with = "deserde_from_str", serialize_with = "serde_to_str")]
     pub private_key: PrivateKey,
     /// passphrase used when import wallet
     pub passphrase: Option<SecUtf8>,
+    /// private_key: public_key-> private_key paire
+    pub key_pairs: BTreeMap<PublicKey, Option<Vec<u8>>>,
```

Comment:
in watch-only it'll be None? or why is it `Option`? and perhaps `SecretVec<u8>`?

Reply:
if hardware wallet, we can't get private key

Reply:
ok, it'll be good to mention in the comment